fili-core/src/test/java/com/yahoo/bard/webservice/druid/model/filter/TestDimensionalFilter.java

```diff
@@ -0,0 +1,29 @@
+// Copyright 2017 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.druid.model.filter;
+
+import com.yahoo.bard.webservice.data.dimension.Dimension;
+
+/**
+ * A dummy test class that simply constructs a dimensional filter.
+ */
+public class TestDimensionalFilter extends DimensionalFilter {
+
```

Comment:
I don't want to add dependency on sub-class to a test on the base class. Inline class was not working, due to some weird groovy error that I think might be a bug.